extending-valtimo/creating-data-provider.md

```diff
@@ -0,0 +1,65 @@
+# What is a data provider
+
+A data provider is a simple way of exposing any kind of data through an API. An engineer can expose the data by writing
+a data provider. 
+
+The data provider setup was created for a specific use-case: the admin of the application should be able to configure a
```

Comment:
Not sure if I like this. Rather than say it's meant for a specific use case, maybe just use it as an example for why you might want to use this.